aikau/src/main/resources/alfresco/util/hashUtils.js

```diff
@@ -113,6 +113,7 @@ define(["dojo/_base/array",
        * a suppresssDecoding argument of true is provided.
        *
        * @instance
+       * @function
```

Comment:
I don't understand how the referencing works, but I've also checked the JSDoc output and it does seem ok - I just wanted to check.